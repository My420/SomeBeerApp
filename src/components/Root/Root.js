import React from 'react';

import { Provider } from 'react-redux';
import AppRouter from '../AppRouter/AppRouter';
import store from '../../store/store';

const Root = () => {
  // eslint-disable-next-line no-console
  console.log('render ===== root');
  return (
    <Provider store={store}>
      <AppRouter />
    </Provider>
  );
};

export default Root;
