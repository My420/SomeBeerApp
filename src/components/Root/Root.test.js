import React from 'react';
import { shallow } from 'enzyme';
import Root from './Root';

describe('Test <Root/> component', () => {
  const wrapper = shallow(<Root />);

  test('should correct render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
