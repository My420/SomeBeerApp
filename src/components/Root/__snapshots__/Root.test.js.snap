// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Root/> component should correct render component 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <AppRouter />
</Provider>
`;
